generators/server/templates/src/main/java/package/config/_OAuth2ServerConfiguration.java

```diff
@@ -85,6 +91,8 @@ public void configure(HttpSecurity http) throws Exception {
             .and()
                 .csrf()
                 .disable()
+                // By default CORS is disabled. Uncomment here and in application.yml to enable.
+                // .addFilterBefore(corsFilter, UsernamePasswordAuthenticationFilter.class)
```

Comment:
So the corsFilter might not be used at all? Then I don't like to have an unused variable around